chore(Card): update a11y card docs (#3384)
patternfly/patternfly-org: packages/v4/patternfly-docs/content/accessibility/card/card.md

| Prop | Applied to | Reason | 
|---|---|---|
| `aria-label="[text describing the card]"` or `aria-labelledby="[id of the element that labels the card]"` | `Card` | Adds an accessible name to the card. If the card has a `CardTitle`, the `aria-labelledby` prop should be passed in with a value of the `CardTitle`'s `id` prop. Otherwise the `aria-label` prop should be passed in to act as a title for the card. |

> | `component="[element tag]"` | `Card` | Sets the base element for the sub-component (`div` element by default). <br/><br/> The `aria-label` or `aria-labelledby` prop should be omitted as these props are not well supported on `div` elements without a `role`. |

Reviewer: Just a slight verbiage tweak:

```suggestion
| `component="[element tag]"` | `Card` | Sets the base element for the sub-component (`div` element by default). <br/><br/> When using `div` as the component, the `aria-label` and `aria-labelledby` props should be omitted as these props are not well supported on `div` elements without a `role`. |
```

Author: Thank you Eric! Great suggestion
